R/prep_scores.R

```diff
@@ -1,7 +1,87 @@
-prep_scores <- function(results_portfolio = NULL, peers_results_aggregated = NULL,
-                        asset_class = "equity") {
-  if (is.null(results_portfolio)) {
-    data_out <- use_toy_data("scores") %>% filter(asset_class == .env$asset_class)
+#' Prepare data input for plotting aggregate climate scores
+#'
+#' @description Prepare data input for plotting aggregate climate scores based
+#' on PACTA for investors output files. These files must have been wrangled with
+#' `prep_data_executive_summary()` before they can be passed to this function.
+#'
+#' @param results_portfolio Data frame that contains pre-wrangled portfolio
+#'   level PACTA results from a PACTA for investors analysis.
+#' @param peers_results_aggregated Data frame that contains pre-wrangled
+#'   aggregate peer group level PACTA results from a PACTA for investors
+#'   analysis.
+#' @param asset_class Character vector of length 1. Must be either `equity` or
+#'   `bonds`.
+#' @param scenario_source Character vector of length 1. Must be a
+#'   `scenario_source` featured in the `scenario_thresholds` data set.
+#'
+#' @return data.frame
+prep_scores <- function(results_portfolio,
+                        peers_results_aggregated,
+                        asset_class,
```

Comment:
nit: if we define asset_class here as: `asset_class = c("equity", "bonds") then we can simplify the check in line 24 to `match.arg(asset_class)`

Reply:
I was wondering yesterday how this worked again, but yes, definitely

Reply:
done